derive_builder/src/options/struct_mode.rs

```diff
@@ -97,6 +124,7 @@ impl From<OptionsBuilder<StructMode>> for (StructOptions, OptionsBuilder<FieldMo
             builder_visibility: m.builder_vis.unwrap_or(m.build_target_vis),
             builder_pattern: b.builder_pattern.unwrap_or_default(),
             build_target_ident: syn::Ident::new(m.build_target_name),
+            derives: m.derive_traits.unwrap_or_default(),
```

Comment:
If we pick 2, we _could_ emit a (deprecation) warning which informs the user that `Debug` and `Clone` derives are already implicit and that deriving them explicitly is redundant? (deprecations and panics are the only ways we can currently communicate with the user).